### DISCUSSION TITLE
文档传输进度一直不变，卡住不动了

### DISCUSSION THREAD

**Discussion by 810-create** [localsend/localsend#1746] [L-简体中文]
![屏幕截图 2024-09-07 154455](https://github.com/user-attachments/assets/5773a405-5b48-4ade-9151-107b2ec49a0d)

**Comment by YuuuDeaNe** [comment]
我现在也是碰到了这种情况

**Comment by TheIce08** [comment]
同问题

**Comment by IvanCyan** [comment]
同样的问题，未解决

**Comment by IvanCyan** [comment]
解决了，似乎不能保存在桌面，我换了D盘就好了

**Reply by unkown-777** [reply]
离谱，果然是这样

**Reply by lingH2O** [reply]
还真是，在桌面新建的文件夹也不行，换到其他盘立马好了

**Reply by xiahy1211** [reply]
好使,谢谢

**Comment by Goal75KG** [comment]
换到非C盘目录即刻见效，怀疑是安装时候的权限问题

**Comment by BigSmart114514** [comment]
那有人交issues吗？ 不过好像没人维护了，已经1年没有更新了

**Reply by Mr-LiuSS** [reply]
看仓库的commit记录，开发者好像在大量重构。
